hw/ip/pattgen/doc/dv/index.md

```diff
@@ -5,36 +5,45 @@ title: "PATTGEN DV document"
 ## Goals
 * **DV**
   * Verify all PATTGEN IP features by running dynamic simulations with a SV/UVM based testbench
-  * Develop and run all tests based on the [testplan](#testplan) below towards closing code and functional coverage on the IP and all of its sub-modules
+  * Develop and run all tests based on the [testplan](#testplan) below 
```

Comment:
For all markdown (.md) files, we need to write one full sentence per line.

Reply:
Wrote one full sentence per line